ddtrace/logger.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+DEFAULT_FILE_SIZE_BYTES = 15 << 20  # 15 MB
+
+
+def configure_ddtrace_logger():
+    # type: () -> None
+    """ """
+    debug_enabled = os.environ.get("DD_TRACE_DEBUG", "false").lower() in ("true", "1")
+    log_path = os.environ.get("DD_TRACE_LOG_FILE", None)
+    max_file_bytes = int(os.environ.get("DD_TRACE_FILE_SIZE_BYTES", DEFAULT_FILE_SIZE_BYTES))
+    num_backup = 1
+    log_format = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s"
+    log_formatter = logging.Formatter(log_format)
+
+    ddtrace_logger = logging.getLogger("ddtrace")
+    ddtrace_logger.propagate = False
```

Comment:
Can we add a comment for this line on why we want `propagate = False`?

Reply:
Let me know if this is what you were looking for: 

When configuring a logger such as `ddtrace = logging.getLogger('ddtrace')`, the parent of the `ddtrace` logger ( obtained via `ddtrace.parent.name`) evaluates to `root`. This ancestor relationship would also apply if a python application opted to create its own logger with the logging library.

Based on the Python Logging Library docs ([link to Python 3.9's version for reference](https://docs.python.org/3.9/library/logging.html#logging.Logger.propagate)), setting `propagate = False` prevents the `ddtrace` logger from affecting `root`. 


Do you mean to leave a comment inside the code? I can do that too and just write something like "# prevent ddtrace logger from affecting root` if that works?

Reply:
> Do you mean to leave a comment inside the code?

This is what I was asking for!

`# prevent ddtrace logger from affecting root`

Maybe

`# prevent ddtrace logger from propagating logs to the root logger`

? too wordy?

Reply:
I'll go with the second version since it's more clear which logger we want to avoid propagating to. Thanks for the suggestion!

Reply:
Resolving this because we got rid of `propagate = False`